### PR TITLE
Removed readOnly from volumeMount

### PR DESCRIPTION
**Pull request by pmmalinov01**
Removed readOnly option from a log file that needs to be written to. And indented small example yaml to be consistant with the bigger example above

### CODE REVIEW

**Review by irvifa — COMMENTED** [review]
/lgtm

File: content/en/docs/tasks/debug-application-cluster/audit.md [hunk 1]
```diff
@@ -156,7 +156,6 @@ volumeMounts:
     readOnly: true
   - mountPath: /var/log/audit.log
     name: audit-log
-    readOnly: false
```
Comment (sftim) [inline content/en/docs/tasks/debug-application-cluster/audit.md:159]: I don't understand this change.

**Review by sftim — COMMENTED** [review]
I think it makes sense to mount the audit log volume read-write.

@pmmalinov01 can you clarify why you recommend this change?

**Comment by sftim** [comment]
@irvifa I hope you don't mind if I
/lgtm cancel

File: content/en/docs/tasks/debug-application-cluster/audit.md (hunk 1 above)
Comment (pmmalinov01) [reply content/en/docs/tasks/debug-application-cluster/audit.md:159]: I have no idea why, but I read it readOnly: true.
Sorry for the noise this PR is not needed.

**Pull request closed without merging**